cachito/web/models.py

```diff
@@ -139,14 +139,19 @@ def from_json(cls, package):
         cls.validate_json(package)
         return cls(**package)
 
-    def to_json(self):
+    def to_json(self, dependencies=None):
```

Comment:
At first I was wondering if it would be worth to receive the request instead of a dependency here and moving the logic to fetch the dependencies in here. But then I couldn't think of a nice way to remove code duplications this would generate. I thought some level of introspection could solve it, but then it would probably break encapsulation and would end up being a bad idea.